Source/Resources/Content/MSGraph/Settings/Group.Unified.json

```diff
@@ -1,4 +1,5 @@
 {
+  "description":"What is it: Configures restrictions for creating Azure AD groups. \n Simeon Notes: Can be used to tighten security around group creation, including naming conventions. Simeon's baseline allows users in the Azure AD group 'BUILTIN - Group Creators' to create groups. \n Learn More: https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide",
```

Comment:
**What does this do?**
Configures restrictions for creating Azure AD Groups. The baseline restricts users not in Group Creators from creating groups. 

**Why should you use this?**
If you want to tighten security around group creation, including naming conventions. 

**What is the end user impact?**
User who are not in Group Creators will be restructured from creating Azure AD groups. 

**Learn more**
[Manage who can create Microsoft 365 Groups](https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide)